packaging/opensuse/snapd.spec

```diff
@@ -203,40 +221,30 @@ export LDFLAGS
 # Generate autotools build system files.
 pushd %{indigo_srcdir}/cmd
 autoreconf -i -f
+
 %configure \
     %{!?with_apparmor:--disable-apparmor} \
+    %{?with_apparmor:--enable-apparmor} \
```

Comment:
If you stay with %bcond_with apparmor, I believe this will need to be refactored info `%if.. %endif`

Reply:
I tried to make the various bcond thing work correctly and I didn't manage to. If you have a patch that makes this what you intend I'm happy to merge it.

Reply:
This is fine, because of how the `with_<FOO>` macros are defined.